man/subscription-manager.8

```diff
@@ -388,15 +376,34 @@ command displays the current configured syspurpose
 .I preferences
 for the system.
 
+.PP
+The
+.B syspurpose
+command has subcommands for all the various syspurpose preferences and attributes:
+
+.IP
+1. addons
+
+.IP
+2. role
+
+.IP
+3. service-level
+
+.IP
+4. usage
+
+
 .TP
 .B --show
 Shows the system's current set of syspurpose preference formatted as JSON. Single-valued entries for which there is no value will be included in the output with a value of "". List entries which have no value will be included in the output with a value of "[]" (less the quotes).
 
 
-.SS ADDONS OPTIONS
+.PP
+.SS addons options
 The
 .B addons
-command displays the current configured addons system purpose attribute
+subcommand displays the current configured addons system purpose attribute
```

Comment:
I would just avoid of using word "subcommand" and I would use just command here.